Pixelels/Collage.py

```diff
@@ -0,0 +1,501 @@
+import pygame, sys, math
+from pygame.locals import *
+
+#key I, G, R, L, W, S, C, P, Q, E, K, Z, F, X
+#Pretty much most the keyboard now
+#Look at the end to see what key relates to which function
+
+Picture = pygame.image.load("PhoenixCol.png") #Just change the image name to use different images
+Picture2 = pygame.image.load("GodotCol.png") # Used for collage
+Running = True
+pygame.init()
+
+PictureScale = 80             # Used in the collage
+PictureScale2 = PictureScale
+PictureScale3 = 0
+ColourManip = 1
+
+
+Width = 1280   #Change these depending on the picture
+Height = 720
+
+Screen = pygame.display.set_mode((Width, Height))
+#Picture =pygame.transform.scale(Picture, (Width,Height))   #Scales the picture to fit the screen
+Picture =pygame.transform.scale(Picture, (PictureScale,PictureScale))     #Used for Collage
```

Comment:
Use inline comments sparingly and try to have comments that explain blocks of code instead.